File: api/python/examples/producer.py
#encoding=utf8
import sys 
sys.path.append('../') 

from zbus import Message, RemotingClient, Producer
 
client = RemotingClient(broker='127.0.0.1:15555')

#Producer是轻量级对象,不需要关闭
p = Producer(client=client, mq='MyMQ') 

msg = Message()    
msg.set_body('hello world') 
print p.send(msg)


client.close()

File: api/nodejs/examples/producer.js
var zbus = require("../zbus");
var Message = zbus.Message;
var MessageClient = zbus.MessageClient;
var Producer = zbus.Producer; 

var client = new MessageClient("127.0.0.1:15555");

client.connect(function(){ 
	var p = new Producer(client, "MyMQ");
    //p.createMQ( function(msg){});
	
    var msg = new Message();
    msg.setBody("hello world from node.js");
    
    p.send(msg, function(res){
        console.log(res.toString());
    }); 
   
});


